llms/mlx_lm/models/nemotron.py

```diff
@@ -0,0 +1,239 @@
+# Copyright © 2024 Apple Inc.
+
+from dataclasses import dataclass
+from typing import Dict, Optional, Tuple, Union
+
+import mlx.core as mx
+import mlx.nn as nn
+
+from mlx_lm.activations import get_activation
```

Comment:
Do you mind to hard code the activation and remove this. It makes the code a little easier to follow if the implementation is self-contained to a file. As far as I know the different models tend to fix the activation so having this flexibility from the config param isn't super useful.

Reply:
Done, thanks for the suggestion!